examples/get_started/02_contour_map.py

```diff
@@ -95,8 +95,8 @@
 # -------------
 #
 # To make it clear where the islands are located, the
-# :meth:`pygmt.Figure.coast` method can be used to color in the land. The
-# ``land`` is colored in as "lightgray", and the ``shorelines`` parameters
+# :meth:`pygmt.Figure.coast` method can be used to color in the landmasses.
```

Comment:
As it is currently not consistent in the documentation, do we want to go with "landmass" or "land-mass"?